phoenix-android/src/main/kotlin/fr/acinq/phoenix/android/utils/datastore/UserPrefs.kt

```diff
@@ -102,19 +103,19 @@ object UserPrefs {
 
     private val TRAMPOLINE_MAX_BASE_FEE = longPreferencesKey("TRAMPOLINE_MAX_BASE_FEE")
     private val TRAMPOLINE_MAX_PROPORTIONAL_FEE = longPreferencesKey("TRAMPOLINE_MAX_PROPORTIONAL_FEE")
-    fun getTrampolineMaxFee(context: Context): Flow<TrampolineFees?> = prefs(context).map {
-        val feeBase = it[TRAMPOLINE_MAX_BASE_FEE]?.let { Satoshi(it) }
-        val feeProportional = it[TRAMPOLINE_MAX_PROPORTIONAL_FEE]
-        if (feeBase != null && feeProportional != null) {
-            TrampolineFees(feeBase = feeBase, feeProportional = feeProportional, cltvExpiryDelta = CltvExpiryDelta(144))
-        } else null
-    }
+
     suspend fun saveTrampolineMaxFee(context: Context, fee: TrampolineFees) = context.userPrefs.edit {
-        it[TRAMPOLINE_MAX_BASE_FEE] = fee.feeBase.sat
-        it[TRAMPOLINE_MAX_BASE_FEE] = fee.feeProportional
+        it[TRAMPOLINE_MAX_BASE_FEE] = fee.feeBase.toLong()
+        it[TRAMPOLINE_MAX_PROPORTIONAL_FEE] = fee.feeProportional
+    }
+
+    fun getTrampolineMaxFee(context: Context): Flow<TrampolineFees> = prefs(context).map {
+        val feeBase = it[TRAMPOLINE_MAX_BASE_FEE] ?: -1L
+        val feeProportional = it[TRAMPOLINE_MAX_PROPORTIONAL_FEE] ?: -1L
+        TrampolineFees(feeBase = Satoshi(feeBase), feeProportional = feeProportional, cltvExpiryDelta = CltvExpiryDelta(144))
```

Comment:
What's the reason for returning an invalid `TrampolineFees` object with negative fees, instead of returning `null` if the preference is missing? It seems more fragile to me.